css/main.css

```diff
@@ -416,6 +417,53 @@ textarea {
     brightness(92%) contrast(82%);
 }
 
+.menu {
+  display: flex;
+  flex-direction: column;
+  color: gray;
+  position: absolute;
+  top: 0;
+  height: 100vh;
+  width: 100%;
+  background-color: black;
```

Comment:
- [ ] As shown in the screenshot below, it'd be better if you could match the **background color** of the mobile menu to the chosen design.
![image](https://user-images.githubusercontent.com/62849872/142594966-d17f9383-c29c-4f49-81a2-673d8fe4ff16.png)

Reply:
- [ ] As the last reviewer said you should stick to design. Kindly change the design.

### **Your design**:
![image](https://user-images.githubusercontent.com/38635158/142631370-fffee33e-3f9a-41f4-9020-7010bf51d010.png)
____

### **Original design:**
![image](https://user-images.githubusercontent.com/38635158/142631722-561f866c-dec1-417b-8bd7-ce98f92de6d0.png)